src/page-modules/assistant/client/journey-planner/index.ts

```diff
@@ -41,14 +46,38 @@ export function useTripPatterns(tripQuery: FromToTripQuery) {
       {},
     );
 
-  const isLoadingInitialData = !data && isLoading;
+  useEffect(() => {
+    if (data) {
+      setNumberOfTripPatterns(
+        data.reduce((acc, curr) => acc + curr.tripPatterns.length, 0),
```

Comment:
Doing a `flatMap` of `data.tripPatterns` and taking the length of that would result in the same number? No problem having it like this either, just me that needs to take a few moments to remember how `reduce` works every time I see it 😅

Reply:
Yes, that would result in the same number 😃 I think both have the same time complexities, but `flatMap` might be easier to read, not sure 🤔 With reduce we don't have to create a new array though, but I guess we don't have a memory issue 😅